packages/website/pages/docs/guides/_meta.json

```diff
@@ -23,6 +23,9 @@
   "enable-a-prover": {
     "title": "✅ Enable a prover"
   },
+  "prover-claim-ttko": {
+    "title": "🛄 Prover Claim TTKO"
```

Comment:
title's should follow sentence case convention. second thing is the phrasing could follow the convention (kind of like how the others start with a verb).. hmm.. maybe "Claim prover TTKO"?

Reply:
Updated to "🛄 Claim prover TTKO".